### PR TITLE
Add neurom.log module with default stream logger

### PR DESCRIPTION
**Pull request by juanchopanza**

### CODE REVIEW

**Comment by juanchopanza** [comment]
Fixes #362.

File: neurom/log.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+* Sets up a default stream handler with shor tformat string and level DEBUG.
+
+* Defines short and long format strings FMT_LONG and FMT_LONG respectively.
+'''
+import logging
+import logging.handlers
+
+# root level logger with stream handler.
+LOG = logging.getLogger()
```
Comment (mgeplf) [inline neurom/log.py]: I would find this behavior quite surprising.  Playing w/ the root logger means that when someone else changes it, importing neurom could change the behavior, right?

Comment (juanchopanza) [inline neurom/log.py]: Yes, good point. I should get a named logger, e.g.

    LOGGER_NAME = 'neurom'
    LOG = logging.getLogger(LOGGER_NAME)

Comment (mgeplf) [inline neurom/log.py]: Yeah, that helps.  I would probably also have a helper to remove it.

**Pull request merged**